autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -657,6 +642,52 @@ def on_graph_execution(
         )
         cls.db_client.send_execution_update(result)
 
+        metadata = cls.db_client.get_graph_metadata(
+            graph_exec.graph_id, graph_exec.graph_version
+        )
+        assert metadata is not None
+        outputs = cls.db_client.get_execution_results(graph_exec.graph_exec_id)
+        logger.info(f"{outputs=}")
```

Comment:
use log_metadata.info

Reply:
But the log seems unnecessary, remove it instead ?

Reply:
The output is unneeded. Good spot